src/EDLog/EDLog.ts

```diff
@@ -358,10 +358,11 @@ export class EDLog extends EventEmitter {
 
     /**
      * Launches the log reader.
-     * @param If true the method will return an array of event logs, otherwise empty.
+     * 
+     * @param backlog Optional configuration settings.
      */
-    public start (backlog: { process: boolean; store: boolean; }): EDEvent[] {
-        fs.watch(this.logReader.getDir(), (eventType, fileName) => {
+    public start(backlog: IBacklogOptions = {}): EDEvent[] {
+        fs.watch(this.logReader.getDir(), (eventType: string, fileName: string) => {
```

Comment:
Wait, why did you add this?

Reply:
Because the original requires an object to be passed in, but an empty object is redundant if no properties are even assigned to that object.

Before:
    
    let log = new EDLog();
    log.start({});

After:

    let log = new EDLog();
    log.start();

The only way to do an optional parameter with specified properties is via an interface. That way you can call it either with an object and intellisense or without any object at all.

Reply:
No, I mean the line below 😄

Reply:
VSCode was complaining that they didn't have types specified. I guess it was before tsc picked up the node bindings, because it seems to not need them now.